frontend/packages/dev-console/src/utils/add-resources-menu-utils.ts

```diff
@@ -38,10 +39,10 @@ export const getAddPageUrl = (
       throw new Error('Invalid Import option provided');
   }
   if (hasApplication && appGroup) {
-    params.append('application', encodeURIComponent(appGroup));
+    params.append(QUERY_PROPERTIES.APPLICATION, encodeURIComponent(appGroup));
```

Comment:
`URLSearchParams` already encodes. The problem is now we're double encoding. Remove all use of `encodeURIComponent` and `decodeURIComponent` when using `URLSearchParams`

Reply:
Removed use of ```encodeURIComponent``` and ```decodeURIComponent```.